server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -192,6 +281,7 @@ export class MongoDbFactory implements core.IDbFactory {
         this.bufferMaxEntries = bufferMaxEntries;
         this.connectionPoolMinSize = connectionPoolMinSize;
         this.connectionPoolMaxSize = connectionPoolMaxSize;
+        this.clientSideRetry = config.facadeLevelRetry;
```

Comment:
nit: Any reason why the original config is called `facadeLevelRetry`, then we assign its value to `clientSideRetry`, which is internally used in `MongoCollection` as `retryEnabled`? The name changes across the flow could be confusing

Reply:
Addressed in b0e2d37

Keep in setting as `facadeLevelRetry` due to it under how mongo setting think it is more explicit. In code, used `retryEnabled` because we know it is for the facade.